docs/root/intro/arch_overview/load_balancing.rst

```diff
@@ -94,10 +94,74 @@ Priority levels
 ------------------
 
 During load balancing, Envoy will generally only consider hosts configured at the highest priority
-level. For each EDS :ref:`LocalityLbEndpoints<envoy_api_msg_LocalityLbEndpoints>` an optional priority
-may also be specified. Currently the failover mechanics for routing from one priority level to
-another are fairly simplistic: a given priority level will be used until it has zero healthy hosts,
-at which point it will hard fail to the next highest priority level.
+level. For each EDS :ref:`LocalityLbEndpoints<envoy_api_msg_LocalityLbEndpoints>` an optional
+priority may also be specifie. When endpoints at the highest priority level (P=0) are healthy, all
+traffic will land on endpoints in that priority leve. As endpoints for the highest priority level
```

Comment:
typo: "leve" - do you mind running through spell checker?

Reply:
Hahaha.  I did spell check, but then I ran a buggy version of check_format which replaced "[^.].  " with ".  " and so removed the last letter of a bunch of words.   Shame on me for not checking diffs before and after.